components/channel/OnSendingData.js

```diff
@@ -0,0 +1,37 @@
+
+import { isBinaryPayload, pascalCase, isStringPayload } from '../../utils/index';
+export function OnSendingData(message, defaultContentType) {
+  const messageName = pascalCase(message.uid());
+  let convertToBinary = '';
+  if (isBinaryPayload(message.contentType(), defaultContentType)) {
+    convertToBinary = `
+      if(beforeSendingHooks.length == 0){
+      dataToSend = Buffer.from(${messageName}}Message.Convert.${messageName}ToJson(dataToSend));
```

Comment:
After first `${messageName}` you have additional `}` This is probably not needed char.

Reply:
Good catch!